solr/core/src/java/org/apache/solr/cli/CreateTool.java

```diff
@@ -192,7 +191,7 @@ protected void createCore(CommandLine cli, SolrClient solrClient) throws Excepti
       }
     } catch (Exception e) {
       /* create-core failed, cleanup the copied configset before propagating the error. */
-      PathUtils.deleteDirectory(coreInstanceDir);
+      org.apache.solr.util.FileUtils.deleteDirectory(coreInstanceDir);
```

Comment:
Should this be a fully qualified package?

Reply:
Unfortunately another FileUtils is used in this file as well